frontend/src/queries/schema/schema-general.ts

```diff
@@ -2528,8 +2528,168 @@ export type CachedTracesQueryResponse = CachedQueryResponse<TracesQueryResponse>
 export interface RevenueTrackingEventItem {
     eventName: string
     revenueProperty: string
+    revenueCurrencyProperty: string | undefined
+}
+
+// NOTE: Keep in sync with posthog/models/exchange_rate/currencies.py
+// to provide proper type safety for the baseCurrency field
+export enum SupportedCurrencies {
```

Comment:
@robbie-c You've had more experience with the codebase, should this be in the singular instead?